### PR TITLE
DW-3985 Add Vars

### PR DESCRIPTION
**Pull request by caseyr232**

### CODE REVIEW

File: terraform/deploy/app/frontendservice.tf [hunk 1]
```diff
@@ -25,11 +25,31 @@ module "ecs-fargate-task-definition" {
...
+      name  = "REACT_APP_OS_URL"
+      value = "https://${data.terraform_remote_state.orchestration-service.outputs.orchestration_service_fqdn}"
+    },
+    {
+      name  = "REACT_APP_REGION"
+      value = data.aws_region.current.name
+    },
+    {
+      name  = "REACT_APP_USERPOOL_WEBCLIENT_ID"
+      value = module.cognito-app.outputs.app_clients.id
```
Comment (IanUtley) [inline terraform/deploy/app/frontendservice.tf:40]: app_clients implies more than 1? Will this choose the first in the list?

File: terraform/modules/cognito-app/outputs.tf [hunk 2]
```diff
@@ -0,0 +1,5 @@
+output outputs {
+  value = {
+    app_clients = aws_cognito_user_pool_client.app_client
```
Comment (IanUtley) [inline terraform/modules/cognito-app/outputs.tf:3]: Ah! This answers my earlier question, can we not name this app_client?

File: terraform/deploy/app/frontendservice.tf [hunk 3]
```diff
@@ -25,11 +25,31 @@ module "ecs-fargate-task-definition" {
   environment = [
     {
       name  = "REACT_APP_UI_LOGIN_URL"
-      value = "https://placeholder-url" # to be amended at a later date (in another ticket)
+      value = "https://${module.ecs-fargate-service.fqdn}"
```
Comment (draagc) [inline terraform/deploy/app/frontendservice.tf:28]: I don't think this is needed anymore as Amplify is handling the auth?

Comment (caseyr232) [reply terraform/deploy/app/frontendservice.tf:28]: @IanUtley ?

File: terraform/deploy/app/frontendservice.tf (hunk 1 above)
Comment (caseyr232) [reply terraform/deploy/app/frontendservice.tf]: Updated

File: terraform/modules/cognito-app/outputs.tf (hunk 2 above)
Comment (caseyr232) [reply terraform/modules/cognito-app/outputs.tf]: Updated

File: terraform/deploy/app/frontendservice.tf (hunk 3 above)
Comment (caseyr232) [reply terraform/deploy/app/frontendservice.tf:28]: Actually just checked and you're right it's not mentioned anywhere else - will remove.

**Pull request merged**